src/client/actions/edit/articleActions.js

```diff
@@ -211,12 +211,22 @@ export const setMentionedItems = (model, items) => {
 
 // YOAST
 export const setSeoKeyword = article => {
-  if (article.get("published")) {
-    const seo_keyword = $("input#edit-seo__focus-keyword").val() || ""
+  // if (article.get("published")) {
```

Comment:
If this commented out code isn't needed anymore let's remove it

Reply:
whoops! my bad I forgot to check this file for stray logs and comments